weatherAplication/Modules/Home/View/HomeViewController.swift

```diff
@@ -81,10 +101,44 @@ class HomeViewController: UIViewController {
 
 }
 
+extension HomeViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return viewModel.lastSearches.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath as IndexPath)
+        cell.textLabel?.textColor = .black
+        cell.backgroundColor = .cyan
+        let lastCityArray = viewModel.lastSearches
+        cell.textLabel?.text = lastCityArray[indexPath.row].city
+        return cell
```

Comment:
jeżeli ta celka ma jakieś customowe kolory etc to może zastanów się nad napisaniem własnej cellki. Nie jest to konieczne ale taka praktyka Ci się przyda

Reply:
☝️